### PR TITLE
tugas andy 4nov

### PR DESCRIPTION
**Pull request by AndyWardhana31**
hallo this is my job, let teach me more about database.

thank you

### CODE REVIEW

File: andy-db.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Database & Common Used Queries 
+    • What is Database
+    //answer
+    Database is a collection of data based on certain conditions that are interconnected so that it is easy to manage it. and also from the management it can be easy to find, store and dispose of information
+
+    • NoSQL vs SQL
```
Comment (iyadztabika) [inline andy-db.txt:6]: can you please explain about what is SQL and NoSQL ? Why and when we use SQL or NoSQL databases ? Thank you

Comment (iyadztabika) [inline andy-db.txt:6]: can you please explain about what is SQL and NoSQL ? Why and when we use SQL or NoSQL databases ? Thank you

File: andy-db.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    // answer 
+    sql and non sql based databases aim to store or manage data, only for sql and non sql there are differences in some characters such as data structure (database type)
+
+
+    • Common used Database Driver (MySQL and Mongo)
```
Comment (iyadztabika) [inline andy-db.txt:13]: Can you explain just a little bit about whay is MySQL and Mongo DB ? And also the advantages and disadvantages about using both of them. Thank you

**Comment by AndyWardhana31** [comment]
Hallo mr indana ,

i have a revisi from my job about sql,nonsql,mysql and mongodb.

**Pull request merged**